src/lib/draw/event_handler.ts

```diff
@@ -10,7 +10,7 @@ export const MapEvents = [
   "dragstart",
 ];
 
-export const DocumentEvents = ["keyPress", "keyUp", "keyDown"];
+export const DocumentEvents = ["keypress", "keyup", "keydown"];
```

Comment:
The bug is that the DOM events aren't camel-cased. And sadly there's no warning or anything if you do `document.addEventListener("typo")`